### DISCUSSION TITLE
Mac version

### DISCUSSION THREAD

**Discussion by RoopakShukla** [aiko-chan-ai/DiscordBotClient#96] [General]
Just wondering when the Mac version of Aiko-chan-ai discord bot client will be released? :)

**Comment by aiko-chan-ai** [comment]
wait ...

**Reply by aiko-chan-ai** [reply]
![image](https://github.com/aiko-chan-ai/DiscordBotClient/assets/71698422/f30466ac-4f1e-4a77-9d82-5d65cea2c0e3)
github ci/cd is too slow

**Comment by aiko-chan-ai** [comment]
try latest

**Reply by RoopakShukla** [reply]
sure thanks , couldnt find it that day sooo

**Reply by Rodaviva29** [reply]
> try latest

Hey Aiko! I can't get the Mac Version to work, tried different versions and keeps saying "Cannot open “DiscordBotClient” because it is damaged. This item should be moved to the Trash."

Usually it say's the app is not trusted and I need to go to Security and Privacy and allow it from there, but it isn't the case. And yes, I'm running MacOS with a M-chip, Apple Silicon.

**Reply by aiko-chan-ai** [reply]
https://github.com/aiko-chan-ai/DiscordBotClient/issues/194#issuecomment-2555817658
